packages/core/src/components/TextField/Styled/InnerWrapper/InnerWrapper.styled.tsx

```diff
@@ -101,6 +101,8 @@ const getHeight = ({ size, theme, minRows, multiline, variant }: InnerWrapperPro
     const smallPadding = 1.1 * 2;
     const medPadding = 1.6 * 2;
 
+    minRows = minRows || 1; // eslint-disable-line no-param-reassign
```

Comment:
@gmukul01 Updated the code. I got a linting error, wondering if it's okay to disable for this line? Or, is there a different approach that might be better?

Reply:
Good call - used rows instead of disabling -  6b2f13a